string buoy_color
---
bool is_right_buoy